docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,572 @@
+
+In our scenario the Core component, and the Optimize importer communicates with a singular Amazon OpenSearch instance. On the other hand, the identity and web-modeler component are connected to seperate Amazon Aurora PostgreSQL instances.
+
+It is assumed that you already have one Amazon OpenSearch instance and two Amazon Aurora PostgreSQL instances setup and ready to go with a username, password, and URL.
+
+#### Connecting to Amazon OpenSearch:
+
+```yaml
+global:
+  elasticsearch:
+    enabled: false
+  opensearch:
+    enabled: true
+    auth:
+      username: user
+      password: pass
+    url:
+      protocol: https
+      host: opensearch.example.com
+      port: 443
+
+elasticsearch:
+  enabled: false
+```
+
+You can see that we have globally disabled all internal component configuration for Elasticsearch through `global.elasticsearch.enabled: false` and also disable internal Elasticsearch through `elasticsearch.enabled: false`.
+
+#### Connecting to External Database for Identity:
+
+Here is how to configure Identity with external Amazon Aurora PostgreSQL:
+
+```yaml
+identity:
+  externalDatabase:
+    enabled: true
+    host: "external-postgres-host"
+    port: 5432
+    username: "identity_user"
+    database: "identity_db"
+    existingSecret: "identity-db-secret"
+    existingSecretPasswordKey: "database-password"
+```
+
+Please make sure to correctly define the protocol, host, and port.
+
+#### Connecting to External Database for Web Modeler:
+
+Here is how to configure Web Modeler with external Amazon Aurora PostgreSQL:
+
+```yaml
+webModeler:
+  externalDatabase:
+    url: "jdbc:postgresql://external-postgres-host:5432/camunda_db"
+    user: "camunda_user"
+    password: "secure_password"
+    existingSecret: "camunda-db-secret"
+    existingSecretPasswordKey: "database-password"
+```
+
+The `existingSecret` can be used to specify an existing Kubernetes secret with the password.
+
+If you would like further information on connecting to external databases, we have a number of guides on doing so with the Camunda Helm Chart:
+
+- [Using existing Elasticsearch](/self-managed/setup/guides/using-existing-elasticsearch.md)
+- [Using Amazon OpenSearch Service](/self-managed/setup/guides/using-existing-opensearch.md)
+  - [Using Amazon OpenSearch Service through IRSA (only applicable if you are running Camunda Platform on EKS)](/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md#opensearch-module-setup)
+- [Running Web Modeler on Amazon Aurora PostgreSQL](/self-managed/modeler/web-modeler/configuration/database.md/#running-web-modeler-on-amazon-aurora-postgresql)
```

Comment:
This could be considered to be moved to line 165 where it mentions `It is assumed that you already have one Amazon OpenSearch instance and two Amazon Aurora PostgreSQL instances setup and ready to go with a username, password, and URL.`

That's also what I mean in the comment [here](https://github.com/camunda/camunda-docs/pull/4703/files#r1909014785).

Probably needs to be rephrased a bit along the lines of `It is expected that you already have one Amazon OpenSearch instance and two Amazon Aurora PostgreSQL instances configured and ready to use, complete with a username, password, and URL. If not, refer to the following guides to help you get started before you continue`.

Reply:
I already mentioned the guides in the prerequisites section. I can just say to refer to the guides there.
Thanks for rephrasing it.